src/helpers/mongodb.js

```diff
@@ -3,34 +3,44 @@ import { LockManager } from 'mongo-locks'
 
 import { config } from '~/src/config'
 
-const mongoPlugin = {
-  name: 'mongodb',
-  version: '1.0.0',
-  register: async function (server) {
-    const mongoOptions = {
-      retryWrites: false,
-      readPreference: 'secondary',
-      ...(server.secureContext && { secureContext: server.secureContext })
+const mongoDb = {
+  plugin: {
+    name: 'mongodb',
+    version: '1.0.0',
+    register: async function (server, options) {
+      server.logger.info('Setting up mongodb')
+
+      const client = await MongoClient.connect(options.mongoUrl, {
+        retryWrites: false,
+        readPreference: 'secondary',
+        ...(server.secureContext && { secureContext: server.secureContext })
+      })
+
+      const databaseName = options.databaseName
+      const db = client.db(databaseName)
+      const locker = new LockManager(db.collection('mongo-locks'))
+
+      await createIndexes(db)
+
+      server.logger.info(`mongodb connected to ${databaseName}`)
```

Comment:
Worth noting your using template literals here. They are only needed if your using variables, otherwise you can just use quotes